tests/Keboola/Extractor/AbstractRedshiftTest.php

```diff
@@ -68,4 +71,10 @@ public function getConfig($driver = 'redshift')
         return $config;
     }
 
+    public function getRedshiftPrivateKey()
+    {
+        // docker-compose .env file does not support new lines in variables so we have to modify the key https://github.com/moby/moby/issues/12997
+        return str_replace('"', '', str_replace('\n', "\n", $this->getEnv('redshift', 'DB_SSH_KEY_PRIVATE')));
```

Comment:
not sure I understand this, I would think `str_replace('\n', '"\n"',` but I guess that it works that way the way you have it?

Reply:
the second one is new line character

Reply:
aha, I see, 👍